projects/cds/src/profiletag/services/profile-tag.injector.service.ts

```diff
@@ -1,31 +1,31 @@
 import { Injectable } from '@angular/core';
 import { merge, Observable } from 'rxjs';
-import { filter, mapTo, switchMap, tap } from 'rxjs/operators';
+import { mapTo, switchMap, tap } from 'rxjs/operators';
 import { CdsBackendConnector } from '../connectors/cds-backend-connector';
+import { ProfileTagLifecycle } from '../model/profile-tag-lifecycle';
 import {
-  CartChangedPushEvent,
-  ConsentChangedPushEvent,
   NavigatedPushEvent,
+  ProfileTagPushEvent,
 } from '../model/profile-tag.model';
+import { ProfileTagPushEventsService } from './profile-tag-push-events.service';
 import { ProfileTagEventService } from './profiletag-event.service';
-import { SpartacusEventService } from './spartacus-event.service';
 
 @Injectable({
   providedIn: 'root',
 })
 export class ProfileTagInjectorService {
   private profileTagEvents$ = this.profileTagEventTracker.getProfileTagEvents();
   private injectionsEvents$: Observable<boolean> = merge(
-    this.notifyProfileTagOfConsentGranted(),
-    this.notifyProfileTagOfCartChange(),
     this.notifyProfileTagOfPageLoaded(),
-    this.notifyEcOfLoginSuccessful()
+    this.notifyEcOfLoginSuccessful(),
+    this.notifyPushEvents(),
+    this.notifyProfileTagOfConsentChanged()
   );
-
   constructor(
     private profileTagEventTracker: ProfileTagEventService,
-    private spartacusEventTracker: SpartacusEventService,
-    private cdsBackendConnector: CdsBackendConnector
+    private profileTagPushEventsService: ProfileTagPushEventsService,
+    private cdsBackendConnector: CdsBackendConnector,
+    private profileTagLifecycleService: ProfileTagLifecycle
```

Comment:
Is this service intended to be exposed to clients for inheriting?
If so, maybe these should be changed to `protected`. If not, then please ignore this comment.

Reply:
So far not needed and can be changed easily later.